src/thermompnn/configs/local_longleaf.yaml

```
@@ -0,0 +1,14 @@
+platform:
+  accel: "gpu"
+  thermompnn_dir: "/proj/kuhl_lab/users/dieckhau/ThermoMPNN"
+
+data_loc:
+        megascale_csv: "/proj/kuhl_lab/users/dieckhau/ThermoMPNN/data/mega_scale/Processed_K50_dG_datasets/Tsuboyama2023_Dataset2_Dataset3_20230416.csv"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 8

(indentation)